test/serdes_test.c

```diff
@@ -222,6 +248,18 @@ static int serialize_EVP_PKEY_legacy_PEM(void **serialized,
     return ok;
 }
 
+static int test_text(const void *data1, size_t data1_len,
```

Comment:
Will these two produce less than useful location output?
Might it be prudent for the calling functions to provide some `TEST_info()` or `TEST_note()` output in addition?

Reply:
Sounds ugly, output wise.  It might be preferable to pass `__FILE__` and `__LINE` to the callbacks and have them call the lower case test functions, which do take exactly those as parameters.

I think that leave that for another PR, and just leave a TODO in this file for now